### PR TITLE
Add uninstalling capacity

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -48,10 +48,10 @@ def tests_createdb(ctx, version):
     db = dbname(version)
     print('Installing database {}'.format(db))
     if is_below_odoo_10(version):
-        ctx.run('odoo.py -d {} --workers=0 --log-level=critical '
+        ctx.run('odoo.py -d {} --workers=0 --log-level=critical -i lunch '
```
Comment (damdam-s) [inline tasks.py]: @guewen dit this to be able to test the uninstall of this module.
If there is a way to to it better, I'm open :smile:

File: tests/test_lyrics_uninstaller.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+from anthem.exceptions import AnthemError
+
+
+def test_uninstall_1():
+    with anthem.cli.Context(None, anthem.cli.Options(test_mode=True)) as ctx:
+        ctx.env['ir.module.module'].search(
+            [('name', '=', 'lunch')]).button_immediate_install()
+        try:
+            uninstall(ctx, [])
+        except AnthemError:
```
Comment (guewen) [inline tests/test_lyrics_uninstaller.py]: You could use `pytest.raises`:

```python
with pytest.raises(AnthemError) as excinfo:
    uninstall(ctx, [])
excinfo.match(r'Cannot uninstall modules.*')
```

File: tests/test_lyrics_uninstaller.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+import anthem.cli
+from anthem.lyrics.uninstaller import uninstall
+from anthem.exceptions import AnthemError
+
+
+def test_uninstall_1():
+    with anthem.cli.Context(None, anthem.cli.Options(test_mode=True)) as ctx:
+        ctx.env['ir.module.module'].search(
+            [('name', '=', 'lunch')]).button_immediate_install()
```
Comment (guewen) [inline tests/test_lyrics_uninstaller.py]: is it required to install the lunch addon in this test?

File: tests/test_lyrics_uninstaller.py (hunk 2 above)
Comment (damdam-s) [inline tests/test_lyrics_uninstaller.py]: ok thanks. I will do it

File: tests/test_lyrics_uninstaller.py (hunk 3 above)
Comment (damdam-s) [inline tests/test_lyrics_uninstaller.py]: hum, I have to install at least a non-core module to test its uninstallation later

Comment (damdam-s) [inline tests/test_lyrics_uninstaller.py]: BTW we can install another one but I found it was the less heavy to load

File: tests/test_lyrics_uninstaller.py (hunk 2 above)
Comment (guewen) [inline tests/test_lyrics_uninstaller.py]: @damdam-s however I goofed in the error message, should be 'You have to provide a list of module.*'

File: tests/test_lyrics_uninstaller.py (hunk 3 above)
Comment (guewen) [inline tests/test_lyrics_uninstaller.py]: In this test you don't test the uninstallation of `lunch` but the call to `uninstall` with an empty list which is expected to fail. So having `lunch` installed or not does not make any difference.

Comment (damdam-s) [inline tests/test_lyrics_uninstaller.py]: Of course. My bad :smile:

**Comment by damdam-s** [comment]
@guewen green ! 
I didn't know how you would like to handle this functionnality (in anthem or in marabunta with a special section) but I needed it for a customer so ...

No problem if you want to to it elsewhere or in another way

**Pull request merged**